src/Record/Record.Test/ImmutableRecordTests.cs

```diff
@@ -47,6 +48,27 @@ public void Record_Does_Not_Have_Provenance()
         result.Should().Throw<RecordException>().WithMessage("Failure in record. A record must have exactly one provenance object.");
     }
 
+
+    [Fact]
+    public void Creating_Record_From_Invalid_JsonLD_Throws()
+    {
+        var invalidJsonLdString = TestData.ValidJsonLdRecordString() + TestData.ValidJsonLdRecordString();
+        var result = () => new Record(invalidJsonLdString);
+        result.Should().Throw<RecordException>();
```

Comment:
```suggestion
        result.Should().Throw<RecordException>().WithMessage("......okokjkjo");
```
But with correct message string